test/integration/etcd_storage_path_test.go

```diff
@@ -874,7 +874,7 @@ func TestEtcd3StoragePath(t *testing.T) {
 */
 
 func testEtcdStoragePath(t *testing.T, etcdServer *etcdtest.EtcdTestServer, getter etcdGetter) {
-	masterConfig, err := testserver.DefaultMasterOptions()
+	masterConfig, err := testserver.DefaultMasterOptionsWithTweaks(false, false)
```

Comment:
We need to start etcd2 only and verify it, so to get better control over the start sequence it's using that path instead of the innate.